lib/install.js

```diff
@@ -396,7 +396,11 @@ function compileUI(done) {
    console.log();
    console.log("compile the web UI");
    shell.pushd("-q", path.join(process.cwd(), "developer", "ab_platform_web"));
-   shell.exec("node node_modules/webpack-cli/bin/cli.js");
+   if (Options.develop) {
+      shell.exec("node node_modules/webpack-cli/bin/cli.js -c webpack.dev.js");
+   } else {
+      shell.exec("node node_modules/webpack-cli/bin/cli.js -c webpack.prod.js");
+   }
```

Comment:
`shell.exec(``node node_modules/webpack-cli/bin/cli.js -c webpack.${Options.develop ? "dev" : "prod"}.js``);`

Should it be better ?